<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <node pkg="mbot_nlu_pytorch" type="mbot_nlu_pytorch_node" name="nlu"
 respawn="false" output="screen" 
 args="$(find mbot_nlu_pytorch)/ros/config/config_mbot_nlu_pytorch.yaml"/>

</launch>